README update

tomsherborne opened the pull request:
Added links to virtual environment setup 
fixed minor grammar problems
Modified POS tags to match spaCy

Commit:
Add virtual environment links
Update POS tags to match spacy

Commit:
Fix grammar errors

andrazhribernik commented on README.md:
```diff
@@ -100,7 +100,7 @@ Example:
+	king-queen==man-woman
 
 
 #### Stop Words
```
Could you please add description for section `Syntax dependecies`. 

Since PR is already open, can you also move these NLP term explanations and examples into a notebook directly as we discussed.

andrazhribernik commented on README.md:
```diff
@@ -15,12 +15,12 @@ Clone this repo from GitHub and open the directory, on a UNIX machine these acti
 	cd pycon-nlp-in-10-lines
 
-We recommend you to install all the dependencies in virtualenv, however this step could be skipped.
+We recommend you to install all the required dependencies in a virtual environment such as [virtualenv](https://virtualenv.pypa.io/en/stable/), however this step could be skipped.
```
Nice!

Commit:
Remove common terms from readme

Commit:
add common terms to notebook

tomsherborne commented:
@andrazhribernik please review move of terms to notebook

Commit:
Add syntax dependencies and edit instructions

Add syntax dependency definition
Fix spelling of instructions
Add image of dependencies
